jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -52,6 +53,10 @@ class UnitConversion(PluginTemplateMixin):
     flux_unit_items = List().tag(sync=True)
     flux_unit_selected = Unicode().tag(sync=True)
 
+    show_translator = Bool(False).tag(sync=True)
```

Comment:
would this ever change during an active session or is this just to disable this capability if this PR is merged before the 3.10 release?

Reply:
I am using `show_translator` for disabling right now. It does have a secondary purpose. When changed to true, an observe is triggered and the correct physical type is selected in the dropdown to start. It may not be best practices to set it this way, if this is done differently and correctly elsewhere I will make the change.

Reply:
ok, if its not easy to remove for now, then we can do this in the follow-up as well.  I suspect we'll either want to listen to AddData events on the spectrum viewer or perhaps just make a call from the parser to set the initial value